lib/alephant/broker/load_strategy/revalidate/refresher.rb

```diff
@@ -0,0 +1,62 @@
+module Alephant
+  module Broker
+    module LoadStrategy
+      module Revalidate
+        class Refresher
+          include Logger
+
+          attr_reader :component_meta
+
+          def initialize(component_meta)
+            @component_meta = component_meta
+          end
+
+          def refresh
+            return if cache.get(inflight_cache_key)
+
+            queue.send_message(component_meta.to_json)
+            cache.set(inflight_cache_key, true)
+          end
+
+          private
+
+          def queue
+            @queue ||= begin
```

Comment:
interesting use of `begin`, as I'd personally have used a `Proc` or a `lambda` here myself. I think because the semantics of `begin` makes this feel a little odd